modules/earthengine-layers/src/earth-engine-mesh-layer.js

```diff
@@ -0,0 +1,253 @@
+/* global createImageBitmap */
+import {CompositeLayer} from '@deck.gl/core';
+import {TileLayer, TerrainLayer} from '@deck.gl/geo-layers';
+import ee from '@google/earthengine';
+import {initializeEEApi} from './ee-api'; // Promisify ee apis
+import {load} from '@loaders.gl/core';
+import {ImageLoader} from '@loaders.gl/images';
+import {deepEqual, promisifyEEMethod} from './utils';
+
+/**
+ * Decoder for Terrarium encoding
+ */
+export const ELEVATION_DECODER = {
+  rScaler: 256,
+  gScaler: 1,
+  bScaler: 1 / 256,
+  offset: -32768
+};
+
+// Global access token, to allow single EE API initialization if using multiple
+// layers
+let accessToken;
+
+const defaultProps = {
+  ...TileLayer.defaultProps,
+  // data prop is unused
+  data: {type: 'object', value: null},
+  token: {type: 'string', value: null},
+  eeObject: {type: 'object', value: null},
+  eeMeshObject: {type: 'object', value: null},
+  visParams: {type: 'object', value: null, equal: deepEqual},
+  // Force animation; animation is on by default when ImageCollection passed
+  animate: false,
+  // Frames per second
+  animationSpeed: 12,
+  refinementStrategy: 'no-overlap'
+};
+
+export default class EarthEngineMeshLayer extends CompositeLayer {
```

Comment:
Now I've deleted the files I thought I already deleted. Referring to copied code, I meant that here the `EarthEngineMeshLayer` class is largely copied from the `EarthEngineLayer` class. I could probably remove a lot of this redundancy by making `EarthEngineMeshLayer` a subclass of `EarthEngineLayer`...